app/src/main/assets/fido2_privileged_allow_list.json

```diff
@@ -72,6 +72,10 @@
           {
             "build": "userdebug",
             "cert_fingerprint_sha256": "19:75:B2:F1:71:77:BC:89:A5:DF:F3:1F:9E:64:A6:CA:E2:81:A5:3D:C1:D1:D5:9B:1D:14:7F:E1:C8:2A:FA:00"
+          },
+          {
+            "build": "release",
+            "cert_fingerprint_sha256": "A8:56:48:50:79:BC:B3:57:BF:BE:69:BA:19:A9:BA:43:CD:0A:D9:AB:22:67:52:C7:80:B6:88:8A:FD:48:21:6B"
```

Comment:
@chirayudesai @t-m-w I extracted this with KeyStoreViewer, I'd be nice if you guys can confirm this to me + the reviewers.

Reply:
This is the correct key for CalyxOS Chromium. Thanks!